src/components/styles/StyledLink.tsx

```diff
@@ -20,3 +21,11 @@ export const StyledLink = (props) => (
 );
 
 export const StyledRouterLink = (props) => <StyledLink as={Link} {...props} />;
+
+export const AbbreviatedRouterLink = styled(StyledRouterLink)`
```

Comment:
naming-wise I might prefer `TruncatedRouterLink` or `ShortenedRouterLink`, abbreviated to me sounds like extracting characters from the string (e.g. like how tablespoon gets abbreviated as tbsp)